glue_vispy_viewers/common/vispy_widget.py

```diff
@@ -60,8 +60,8 @@ def __init__(self, parent=None):
         # we support, but if we support more in future, we should implement
         # that here
 
-        # Remove the fov=60 here to solve the mismatch of selection problem
-        self.view.camera = scene.cameras.TurntableCamera(parent=self.view.scene, fov=60, distance=2.0)
+        # orthographic perspective view as default
```

Comment:
Since you need to make a small commit to restart Travis/AppVeyor, just change ``orthographic`` to ``Orthographic`` :)